pkg/agent/core/ngt/handler/grpc/index_test.go

```diff
@@ -1061,6 +1061,31 @@ func Test_server_CreateAndSaveIndex(t *testing.T) {
 		}
 		return nil
 	}
+
+	/*
+		- Equivalence Class Testing (with copy on write disable)
+			- case 1.1: success to create and save 1 uncommitted insert index
+			- case 1.2: success to create and save 100 uncommitted insert index
+			- case 2.1: success to create and save 1 uncommitted delete index
+			- case 2.2: success to create and save 100 uncommitted delete index
+			- case 3.1: success to create and save 1 uncommitted update index
+			- case 3.2: success to create and save 100 uncommitted update index
+		- Boundary Value Testing
+			- case 1.1: fail to create and save 0 index
+			- case 2.1: success to create and save index with invalid dimension
+				- the invalid index will be removed from NGT and the index file
+		- Decision Table Testing
+			- case 1.1: success to create and save index with in-memory mode
+				- do nothing and no file will be created
+			- case 2.1: success to create and save 1 inserted index with copy-on-write enabled
+			- case 2.2: success to create and save 100 inserted index with copy-on-write enabled
```

Comment:
how much uncommitted indexes do you implement with?

Reply:
it is written in the test case name, in case 2.1 it is 1 uncommitted index and case 2.2 is 100.

Reply:
how about 1.1?

Reply:
Sorry I missed it. How about it?
```suggestion
			- case 1.1: success to create and save 100 index with in-memory mode
				- do nothing and no file will be created
			- case 2.1: success to create and save 1 inserted index with copy-on-write enabled
			- case 2.2: success to create and save 100 inserted index with copy-on-write enabled
```